test/e2e/cypress/integration/share-file.spec.ts

```diff
@@ -1,28 +0,0 @@
-import * as path from 'path';
```

Comment:
Why is this test deleted?

Reply:
Because it doesn't work and I already check that feature in the new test `download-shared-file.spec.ts`. Where I first share the file and then with the copied shared link I download it. 
If you consider it more appropriate I could separate it.

Reply:
Nah, for me it's okay to delete it then 🙆🏼‍♂️